.github/workflows/qasp-owasp-scan.yml

```diff
@@ -0,0 +1,38 @@
+name: Run OWASP scan when a PR moves into QASP Review
+on:
+  pull_request:
+    types:
+    - labeled
+    - synchronize
+    - reopened
+# Prevent running more than one job at a time per branch.
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+jobs:
+  qasp_label_check:
+    name: Initiate OWASP scan in Circle CI
+    env:
+      # Check the label from the pull request itself, rather than the event -
+      # not all event types will contain the label name.
+      HAS_QASP_LABEL: ${{ contains(github.event.pull_request.labels.*.name, 'QASP Review') }}
```

Comment:
Adding this to the job env allows us to reuse the value and skip a step down below to export it across steps.